### ISSUE TITLE
doctest dependencies don't seem to be taken into account

### ISSUE THREAD

**Issue by nmattia** [est31/cargo-udeps#52]
I _believe_ this is the case. In particular I have a crate with a doctest that uses `serde_json`:

``` rust
/// ```
/// use serde_json;
/// ... (use serde_json here)
```

and cargo udeps reports the crate as not being used:

```
my_crate v0.1.0 (/path/to/my_crate)`
└─── (dev-)dependencies
     └─── "serde_json"
```

However `cargo test` (correctly) fails if I remove the crate.

(note: I'm running `cargo udeps --all-targets`).

**Comment by est31** [comment]
Yes, it's a known issue: https://github.com/est31/cargo-udeps/blob/6e0b23342f8b6903f1daedb21b5134e11a5fd513/src/lib.rs#L413

**Comment by nmattia** [comment]
Thanks for the quick response! Anything we can do about it?

**Comment by est31** [comment]
#4 could solve it. But it's a larger project.
